ServiceStation/ConsoleApp/Runner.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace ConsoleApp
+{
+    public class Runner

```

Comment:
Why do you need this?)

Reply:
For a service created by a dependency injection (AddTransient method in Startup)

Reply:
Where this service used?)

Reply:
I see only registration, but not usage.

Reply:
Startup class, line 22

Reply:
Its a registration, but where the usage?)))